### PR TITLE
Feb20 Heartbeat draft

### PR DESCRIPTION
**Pull request by andronovhopf**

### CODE REVIEW

**Comment by andronovhopf** [comment]
Not sure what's up with circleci :/

File: content/blog/2020-02-10-february-20-dvc-heartbeat.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  - Heartbeat
+  - Continuous integration
+  - DVC
+---
+
+Hello, friends! Happy February. Just in time for Valentine's day, here's a
+seasonally-relevant DVC pipeline
+[generated by one of our users](https://medium.com/nlp-trend-and-review-en/use-dvc-to-version-control-ml-dl-models-bef61dbfe477).
+
+![](/uploads/images/2020-02-10/pipeline_heart.png)
```
Comment (shcheklein) [inline content/blog/2020-02-10-february-20-dvc-heartbeat.md]: should it be part of the next issue?

Comment (shcheklein) [reply content/blog/2020-02-10-february-20-dvc-heartbeat.md]: if we keep it here - let's make it main picture? edit a little bit (put a real heart line on top? and DVC name?) - it'll be way better to have this then the current main generic image

File: content/blog/2020-02-10-february-20-dvc-heartbeat.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+[Saugat Pachhai](https://github.com/skshetry) and data scientist
+[Elle O'Brien](https://twitter.com/andronovhopf). Saugat, based in Nepal, will
+be contributing to core DVC. Elle (that's me!), currently in San Francisco, will
+be leading data science projects and outreach with DVC.
+
+We're **gearing up for a spring full of talks** about DVC projects, including
+new up-and-coming features for data cataloging and continuous integration. Here
+are just a few events that have been added to our schedule:
+
+-Engineer [Pavel Redzynski](https://twitter.com/Paffciu1) was accepted to speak
```
Comment (shcheklein) [inline content/blog/2020-02-10-february-20-dvc-heartbeat.md]: problem with the lists

Comment (shcheklein) [reply content/blog/2020-02-10-february-20-dvc-heartbeat.md]: can we put embeds for this news items?

Comment (shcheklein) [reply content/blog/2020-02-10-february-20-dvc-heartbeat.md]: or we can put an image with these dates - like calendar or something?

File: content/blog/2020-02-10-february-20-dvc-heartbeat.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+<external-link
+href="https://blog.codecentric.de/en/2020/01/remote-training-gitlab-ci-dvc/"
+title="Remote training with GitLab-CI and DVC"
+description="by Marcel Mikl and Bert Besser"
+link="blog.codecentric.de"
+image="/uploads/images/2020-02-10/marcel.png" />
+
+Here are a few more stories on our radar:
+
+--**AI Singapore shares their method for AI development and deployment.** This
```
Comment (shcheklein) [inline content/blog/2020-02-10-february-20-dvc-heartbeat.md]: lists are not properly rendered

File: content/blog/2020-02-10-february-20-dvc-heartbeat.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+> the network with the same internet connexion."
+
+Thanks for sharing- this is a lovely result. Oh, and last...
+
+--**DVC is a job requirement**! We celebrated a small milestone when we stumbled
+across a listing for a data engineer to support R&D at
+[Elvie](https://www.elvie.com/en-us/), a maker of tech for women's health
+(pretty neat mission). The decorations on the job posting are ours 😎
+
+![](/uploads/images/2020-02-10/elvie.png)
```
Comment (shcheklein) [inline content/blog/2020-02-10-february-20-dvc-heartbeat.md]: put subtitles and alts for images where appropriate ... for subtitles include links when it's relevant

File: content/blog/2020-02-10-february-20-dvc-heartbeat.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+  community takeaways, and everything else along the way.
+  Look here for updates about DVC, our journey as a startup, 
+  projects by our users and big ideas about best practices in ML and data science.
+picture: ../../static/uploads/images/2020-02-10/Heartbeat_February.png
+pictureComment:
+author: ../authors/elle_obrien.md
+commentsUrl: https://discuss.dvc.org/t/january-20-dvc-heartbeat/314
+tags:
+  - Heartbeat
+  - Continuous integration
```
Comment (shcheklein) [inline content/blog/2020-02-10-february-20-dvc-heartbeat.md]: inegration -> Integration?

**Comment by shcheklein** [comment]
@andronovhopf also, please merge master - it should fix the CI issue

File: content/blog/2020-02-10-february-20-dvc-heartbeat.md (hunk 1 above)
Comment (andronovhopf) [reply content/blog/2020-02-10-february-20-dvc-heartbeat.md]: Normally yes, but I don't think it's effective to have Valentine's Day content in March. Going to break the rules slightly to put it here. I'll edit it to make it the cover photo

File: content/blog/2020-02-10-february-20-dvc-heartbeat.md (hunk 2 above)
Comment (andronovhopf) [reply content/blog/2020-02-10-february-20-dvc-heartbeat.md]: I like the idea of embeds, let me see what I can do

Comment (andronovhopf) [reply content/blog/2020-02-10-february-20-dvc-heartbeat.md]: @shcheklein OK i added embeds; let me know what you think of this

**Pull request merged**